### PR TITLE
Release v0.10.10.post1

### PR DESCRIPTION
**Pull request by angrybayblade**
## Release summary

Version number: v0.10.10.post1

## Release details

Autonomy:
- Update `tendermint` testnet command generator to accommodate for new container naming format 

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side), from `develop`
- [ ] I've updated the dependencies versions to the latest, wherever is possible.
- [ ] Lint and unit tests pass locally (please run tests also manually, not only with `tox`)
- [ ] I built the documentation and updated it with the latest changes
- [ ] I've added an item in `HISTORY.md` for this release
- [ ] I bumped the version number in the `__init__.py` file.
- [ ] I published the latest version on TestPyPI and checked that the following command work:
       ```pip install project-name==<version-number> --index-url https://test.pypi.org/simple --force --no-cache-dir --no-deps```
- [ ] After merging the PR, I'll publish the build also on PyPI. Then, I'll make sure the following
      command will work:
      ```pip install project-name-template==<version_number> --force --no-cache-dir --no-deps```  
- [ ] After merging the PR, I'll tag the repo with `v${VERSION_NUMVER}` (e.g. `v0.1.2`)

## Further comments

Write here any other comment about the release, if any.

### CODE REVIEW

File: tests/test_base.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def get_test_files(package_type: PackageType) -> List[Path]:
 
 def test_version() -> None:
     """Test the version."""
-    assert autonomy.__version__ == "0.10.10"
+    assert autonomy.__version__ == "0.10.10.post1.post1"
```
Comment (Adamantios) [inline tests/test_base.py:65]:
```suggestion
    assert autonomy.__version__ == "0.10.10.post1"
```

File: autonomy/deploy/generators/docker_compose/base.py [hunk 2]
```diff
@@ -215,13 +215,11 @@ def generate_config_tendermint(self) -> "DockerComposeGenerator":
...
-                [
-                    f"--hostname=node{k}"
-                    for k in range(self.service_builder.service.number_of_agents)
-                ]
-            ),
+        hosts = " \\\n".join(
+            [
+                "--hostname=" + self.service_builder.get_tm_container_name(index=k)
+                for k in range(self.service_builder.service.number_of_agents)
+            ]
```
Comment (Adamantios) [inline autonomy/deploy/generators/docker_compose/base.py:222]: Can you explain how this resolves the issue?

Comment (angrybayblade) [reply autonomy/deploy/generators/docker_compose/base.py:222]: Before the fix the hostnames were being formatted as `--hostname=node{NODE_ID}`, with this change they'll be formatted as `--hostname={SERVICE_NAME}_tm_{NODE_ID}`

**Review by dvilelaf — COMMENTED** [review]
Has this been tested against an actual service? For example, the hello world or simple abci service?

File: autonomy/deploy/generators/docker_compose/base.py (hunk 2 above)
Comment (Adamantios) [reply autonomy/deploy/generators/docker_compose/base.py]: I understand this part. I do not understand the situation though. I do not know what issue this PR is trying to solve. The description does not help. Is it simply something that was missed on #2016? What is the symptom of the issue? Why was `hosts` a `tuple` and now it is a string?

**Review by dvilelaf — APPROVED** [review]
Just run this with IEKit. Can confirm it seems to be working now.

File: autonomy/deploy/generators/docker_compose/base.py (hunk 2 above)
Comment (angrybayblade) [reply autonomy/deploy/generators/docker_compose/base.py]: So, when building deployments that uses pre-established tendermint networks we build a testnet using `tendermint testnet` command and use this a said pre-established tendermint network. Here when defining the number of validators we also define the hostnames for every node using `--hostname=<NAME>`. And this was missed during the last release.

**Review by Adamantios — APPROVED** [review]

**Pull request merged**